src/main/java/Event.java

```diff
@@ -0,0 +1,24 @@
+public class Event extends Task{
+    String at;
+
+    public Event(String task, String at){
+        super(task);
```

Comment:
You might want to consider replacing 'task' with 'description' to avoid confusion.